routes/web.php

```diff
@@ -10,6 +10,11 @@
 | and give it the controller to call when that URI is requested.
 */
 
+// test
+Route::get("/mollie/test", function () {
+    dd(Mollie::api()->payments->page());
```

Comment:
Is dit deel van de definitieve PR of ben je dit vergeten te verwijderen? In het eerste geval zou ik dit liever op een meer gestructureerde plek binnen de routes zien.

Reply:
Deze ben ik vergeten weg te halen. ik zal hem weg halen